tests/cmdline/utils/test_decorators.py

```diff
@@ -67,3 +69,26 @@ def test_load_backend_if_not_loaded_with_loaded_profile(config, manager):
     # Calling the method again should keep the currently loaded profile, and not switch to the default profile
     load_backend_if_not_loaded()
     assert manager.get_profile().name == 'profile-two'
+
+
+def test_load_backend_if_not_loaded_load_once(manager, monkeypatch):
+    """Test the :meth:`aiida.cmdline.utils.decorators.load_backend_if_not_loaded` calls load profile only once."""
```

Comment:
Fair that it could be more precise but I think it should state then:
```python
"""Test that :meth:`aiida.cmdline.utils.decorators.load_backend_if_not_loaded` calls `Manager.get_profile_storage` only once."""
```
At least that is the thing that we are actively testing.